docs/content/develop/run-tests.md

```diff
@@ -1,53 +1,182 @@
 ---
-title: Use built provider
-weight: 40
+title: "Run tests"
+weight: 70
 aliases:
+  - /docs/getting-started/run-provider-tests
   - /docs/getting-started/use-built-provider
+  - /get-started/run-provider-tests
+  - /get-started/use-built-provider
+  - /getting-started/run-provider-tests
   - /getting-started/use-built-provider
 ---
 
-# Use built provider locally
+# Run tests
 
-{{< hint info >}}
-**Note:** If you want to test changes you've made in Magic Modules, you need to first [generate](/magic-modules/docs/getting-started/generate-providers/) the provider you want to test.
-{{< /hint >}}
+## Before you begin
+
+1. [Generate the modified provider(s)](/magic-modules/docs/getting-started/generate-providers/)
+
+
+1. Set up application default credentials for Terraform
+
+    ```bash
+    gcloud auth application-default login
+    export GOOGLE_USE_DEFAULT_CREDENTIALS=TRUE
+    ```
+
+1. Set the following environment variables:
+
+    ```bash
+    export GOOGLE_USE_DEFAULT_CREDENTIALS=TRUE
+    export GOOGLE_PROJECT=PROJECT_ID
+    export GOOGLE_REGION=us-central1
+    export GOOGLE_ZONE=us-central1-a
+    ```
+
+1. Optional: Some tests may require additional variables to be set, such as:
+
+    ```
+    GOOGLE_ORG
+    GOOGLE_BILLING_ACCOUNT
+    ```
+
+{{< tabs "version" >}}
+
+{{< tab "GA Provider" >}}
+
+## Run automated tests
+
+Run unit tests and linters
+
+```bash
+cd $GOPATH/src/github.com/hashicorp/terraform-provider-google
+make test
+make lint
+```
+
+## Run acceptance tests
+
+Run acceptance tests
```

Comment:
What would you think about linking to Go's documentation about `-run` instead of having the following two paragraphs?

```suggestion
2. Run acceptance tests for only modified resources. (Full test runs can take over 9 hours.) See Go's documentation for more information about `-run`.
```